site/assets/js/snippets.js

```diff
@@ -82,9 +82,19 @@
 
   function alert(message, type, typeVisuallyHidden) {
     const wrapper = document.createElement('div')
-    wrapper.innerHTML = `<div class="alert alert-${type} alert-dismissible" role="alert"><span class="alert-icon"><span class="visually-hidden">${typeVisuallyHidden}</span></span><p>${message}</p><button type="button" class="btn-close" data-bs-dismiss="alert"><span class="visually-hidden">Close</span></button></div>`
+    wrapper.innerHTML = `<div class="alert alert-${type} alert-dismissible" role="alert">
+        <span class="alert-icon"><span class="visually-hidden">${typeVisuallyHidden}</span></span><p>${message}</p>
+        <button type="button" class="btn-close" data-bs-dismiss="alert" data-bs-toggle="tooltip" data-bs-placement="bottom" data-bs-title="Close" data-bs-id="btn-close">
+        <span class="visually-hidden">Close</span></button></div>`
 
     alertPlaceholder.append(wrapper)
+
+    // Create btn-close's tooltip after innerHTML has been modified
+    const tooltip = new boosted.Tooltip(document.querySelector('[data-bs-id="btn-close"]'))
+    // Hide tooltip when clicking on live alert's btn-close
+    document.querySelector('[data-bs-id="btn-close"]').addEventListener('click', () => {
+      tooltip.hide()
+    })
```

Comment:
Needed to add some js to manage tooltip on live alert's close button, since the innerHTML is added after global tooltips' initialisation. (see [http://https://deploy-preview-1823--boosted.netlify.app/docs/5.2/components/alerts/#live-example](http://https//deploy-preview-1823--boosted.netlify.app/docs/5.2/components/alerts/#live-example)) for showing and hiding